apps/docs/content/guides/platform/your-monthly-invoice.mdx

```diff
@@ -0,0 +1,77 @@
+
+<Image
+  alt="Example Invoice"
+  src={{
+    light: '/docs/img/guides/platform/example-invoice.png',
+    dark: '/docs/img/guides/platform/example-invoice.png',
+  }}
+  zoomable
+/>
+
+1) The final amount due
+2) Fixed subscription fee for the current billing cycle
+3) Usage based fee for Compute for the previous billing cycle. There were two projects (wsmmedyqtlrvbcesxdew, wwxdpovgtfcmcnxwsaad) running 744 hours (24 hours * 31 days). These projects incurred $10 in Compute fees each. With $10 in Compute Credits deducted, the final Compute fees are $10.
+4) Usage based fee for Custom Domain for the previous billing cycle. There is no free usage quota for Custom Domain. You get charged for the 744 hours (24 hours * 31 days) a Custom Domain was active. The final Custom Domain fees are $10.19.
+5) Usage based fee for Egress for the previous billing cycle. There is a free usage quota of 250 GB for Egress. You get charged for usage beyond 250 GB only, meaning for 2,119.47 GB. The final Egress fees are $190.75.
+6) Usage based fee for Monthly Active Users for the previous billing cycle. There is a free usage quota of 100,000 users. With 141 users there is no charge for this line item.
+
+### Why is your invoice more than $25?
+The amount due of your invoice being higher than the $25 subscription fee for the Pro Plan can have several reasons.
+
+- **Running several projects:** You had more than one project running in the previous billing cycle. Supabase provides a dedicated server and database for every project. That means that every project you launch incurs compute costs. While the $10 Compute Credits cover a single project using a Nano or Micro compute instance, every additional project adds at least $10 compute costs to your invoice.
+- **Usage beyond quota:** You exceeded the included usage quota for one or more line items in the previous billing cycle while having the Spend Cap disabled.
+- **Usage that is not covered by the Spend Cap:** You had usage in the previous billing cycle that is not covered by the Spend Cap. For example using an IPv4 address or a custom domain.
+
+## How to settle your invoices
+Monthly invoices are auto-collected by charging the payment method marked as "active" for an organization.
+
+### Payment failure
+If your payment fails, Supabase retries the charge several times. We send you a Payment Failure email with the reason for the failure. Follow the steps outlined in this email. You can manually trigger a charge at any time via
+- the link in the Payment Failure email
+- the "Pay Now" button on the [organization invoices](https://supabase.com/dashboard/org/_/invoices) page
+
+
+## Where to find your invoices
+Your invoice is sent to you via email on payment success. You can also find your invoices on the [organization invoices](https://supabase.com/dashboard/org/_/invoices) page.
```

Comment:
I think the invoice is sent once issued and not on payment success

Reply:
I double checked the Orb docs. Seems to happen on payment success (for invoices that are auto-collected). I removed this info completely though. I think it's not really relevant.

> Invoice issued: delivered as soon as the invoice transitions to the issued state. Note that if the invoice is immediately auto-collected (i.e. it’s due on issue), this email will not be sent in lieu of a payment success or failure email. This email type can also be triggered manually via the Orb web application.